bin/import_cv_personal.py

```diff
@@ -0,0 +1,219 @@
+        rows += processed[1]
+        bar.update(i)
+    bar.update(num_samples)
+    pool.close()
+    pool.join()
+
+    imported_samples = get_imported_samples(counter)
+    assert counter["all"] == num_samples
+    assert len(rows) == imported_samples
+    print_import_report(counter, SAMPLE_RATE, MAX_SECS)
+
+    output_csv = os.path.join(os.path.abspath(audio_dir), "data.csv")
+    print("Saving new Coqui STT-formatted CSV file to: ", output_csv)
+    with open(output_csv, "w", encoding="utf-8", newline="") as output_csv_file:
+        print("Writing CSV file for train.py as: ", output_csv)
+        writer = csv.DictWriter(output_csv_file, fieldnames=FIELDNAMES)
+        writer.writeheader()
+        bar = progressbar.ProgressBar(max_value=len(rows), widgets=SIMPLE_BAR)
+        for filename, file_size, transcript, _ in bar(rows):
+            if space_after_every_character:
+                writer.writerow(
+                    {
+                        "wav_filename": filename,
+                        "wav_filesize": file_size,
+                        "transcript": " ".join(transcript),
+                    }
+                )
+            else:
+                writer.writerow(
+                    {
+                        "wav_filename": filename,
+                        "wav_filesize": file_size,
+                        "transcript": transcript,
+                    }
+                )
+    return rows
+
+
+def _preprocess_data(tsv_file, audio_dir, space_after_every_character=False):
+    set_samples = _maybe_convert_set(tsv_file, audio_dir, space_after_every_character)
+
+
+def _maybe_convert_wav(mp3_filename, wav_filename):
+    if not os.path.exists(wav_filename):
+        transformer = sox.Transformer()
+        transformer.convert(samplerate=SAMPLE_RATE, n_channels=CHANNELS)
+        try:
+            transformer.build(mp3_filename, wav_filename)
+        except sox.core.SoxError:
+            pass
+
+
+def parse_args():
+    parser = get_importers_parser(
+        description="Import Common Voice data from a single user's account"
+    )
+    parser.add_argument("tsv_file", help="Path to the single TSV file")
+    parser.add_argument("zip_file", help="Zipped directory containing MP3 clips")
+    parser.add_argument(
+        "--filter_alphabet",
+        help="Exclude samples with characters not in provided alphabet",
+    )
+    parser.add_argument(
+        "--normalize",
+        action="store_true",
+        help="Converts diacritic characters to their base ones",
+    )
+    parser.add_argument(
+        "--space_after_every_character",
+        action="store_true",
+        help="To help transcript join by white space",
+    )
+    return parser.parse_args()
+
+
+def main():
+
+    with zipfile.ZipFile(os.path.abspath(PARAMS.zip_file), "r") as zipped:
+        zipped.extractall(os.path.abspath(os.path.dirname(PARAMS.zip_file)))
+
+    audio_dir = os.path.splitext(os.path.abspath(PARAMS.zip_file))[0]
```

Comment:
style nit: Python's destructuring assignment lets you make the semantics clearer here:
```suggestion
    audio_dir, _ = os.path.splitext(os.path.abspath(PARAMS.zip_file))
```
Already tells me audio_dir is the first value of a tuple, before I even read `os.path.splitext`.